### PR TITLE
Track dependencies in Makefile

### PR DESCRIPTION
**Pull request by AntonioND**
This is an alternative solution to https://github.com/rednex/rgbds/pull/424

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,6 +112,15 @@ rgbfix: ${rgbfix_obj}
 rgbgfx: ${rgbgfx_obj}
 	$Q${CC} ${REALLDFLAGS} ${PNGLDFLAGS} -o $@ ${rgbgfx_obj} ${PNGLDLIBS}
 
+# Dependencies
+rgbasm_deps	:= ${rgbasm_obj:.o=.d}
```
Comment (ISSOtm) [inline Makefile]: I thought substitutions like this weren't supported by BSD Make?

**Comment by ISSOtm** [comment]
Isn't this an alternative to #428, rather?

File: Makefile (hunk 1 above)
Comment (AntonioND) [reply Makefile]: Oh, it didn't occur to me that it could be the case. Honestly, no idea.

**Comment by AntonioND** [comment]
> Isn't this an alternative to #428, rather?

Yes. In any case, it looks like this isn't BSD make, so I'll close it. In case we decide to stop using it, at least it's here for reference.

**Comment by ISSOtm** [comment]
Agreed.

**Pull request closed without merging**